phpstan.neon

```diff
@@ -0,0 +1,4 @@
+parameters:
+    level: 3
```

Comment:
I liked the -l param more as that one is also visible in the output.
This one is hidden away and more difficult to find, if you just want to know the level.
But no biggie.